test/configure.mjs

```diff
@@ -0,0 +1,139 @@
+import { spawner } from './helpers/index.js'
+
+await spawner(
+  function (test) {
+    test.configure({ source: true })
+
+    test('classic test', function (t) {
+      t.fail('custom')
+    })
+  },
+  `
+  TAP version 13
+
+  # classic test
+      not ok 1 - custom
+        ---
+        operator: fail
+        stack: |
```

Comment:
Source wasn't being showed before, and still doesn't for some reason, something related that is a eval instead of a file

Reply:
Unrelated to the PR tho, but the test is not that useful if it doesn't outputs the source

Reply:
yea lets the kill the test then